mesos-client/src/main/scala/com/mesosphere/mesos/conf/MesosClientSettings.scala

```diff
@@ -8,7 +8,8 @@ case class MesosClientSettings(conf: Config) {
   val master: String = conf.getString("connection.masterUrl")
   val redirectRetries: Int = conf.getInt("connection.redirectRetries")
   // we want FiniteDuration, the conversion is needed to achieve that
-  val idleTimeout: FiniteDuration = Duration.fromNanos(conf.getDuration("connection.idleTimeout").toNanos)
+  val idleTimeout: FiniteDuration =
+    Duration.fromNanos(conf.getDuration("connection.idleTimeout").toNanos)
```

Comment:
is this a line length issue?   160 should cover this right?

Reply:
Are 160 characters enough for everybody? ^.^

Reply:
It's too much but I'm probably alone with that. 120 makes reviewing on Github much easier. 160 is ok. Everything above makes it hard to review on MacBooks IMHO.

Reply:
120 please 🙏